### ISSUE TITLE
Rename floating-focus-a11y to floating-focus

### ISSUE THREAD

**Issue by guidobouman** [Q42/floating-focus-a11y#42]
The package name `floating-focus` is available on both NPM and here on Github as they're namespaced behind Q42.

**Comment by CptMeatball** [comment]
This really ties into what the purpose of this package should be. If we want to make this is a general purpose library, we could consider deprecating the `@q42/floating-focus-a11y` and renaming it to `@q42/floating-focus`. If we say our goal is to provide an easy to use accessibility package, the `a11y` postfix is not that bad. 

Either way, renaming the package is not something that `npm` supports, so we have to deprecate the package and publish a new one. I'm not sure that's worth the trouble.

Do you have an opinion regarding this @Jirinrin ?

**Comment by Jirinrin** [comment]
yaaa I also think the a11y suffix doesn't feel that bad, as it does communicate the reason you might want to use this package. I guess it might be a lil easier to find without it, but I don't really think going through the trouble is worth it.

**Comment by Jirinrin** [comment]
Closing this issue for now
